slsim/lsst_science_pipeline.py

```diff
@@ -402,3 +404,77 @@ def add_object(dp0_image, objects, calibFluxRadius=12):
             )
             injected_image = np.array(dp0_image.image.array) + np.array(convolved_image)
             return injected_image
+
+
+def cutout_image_psf_kernel(
+    dp0_image, lens_class, band, mag_zero_point, delta_pix, num_pix, calibFluxRadius=12
+):
+    """This function extracts psf kernels from the dp0 cutout image at point source
+    image positions and deflector          position.
```

Comment:
do I understand this right and here we have different PSF's for the image position and deflector position? How do we deal with this in the image simulation? Perhaps a bit more description desired

Reply:
Yes, PSF kernel in dp0 data vary with points and we can compute PSF kernels at different points using given PSF model.